src/components/ComputeModal.js

```diff
@@ -101,6 +93,34 @@ const DataprocDiskSelector = ({ value, onChange }) => {
   ])
 }
 
+const SparkInterface = ({ interfaceName, synopsis, namespace, name, onDismiss }) => {
+  const interfaceDisplayName = Utils.switchCase(interfaceName,
+    ['yarn', () => 'YARN Resource Manager'],
+    ['apphistory', () => 'YARN Application Timeline'],
+    ['hdfs', () => 'HDFS NameNode'],
+    ['sparkhistory', () => 'Spark History Server'],
```

Comment:
Is this how it's supposed to look? It suggests we have an invalid path configuration, not enough context to confirm/deny (cc @rtitle ) 
![image](https://user-images.githubusercontent.com/6465084/135642860-56830418-a695-4ad3-8599-0e41d0f78915.png)

Reply:
Good question @jdcanas! I had assumed that output was because I didn't run anything on my cluster but I hadn't noticed the question there:

> Did you specify the correct logging directory?

@rtitle I see that the [Spark Console Design Doc](https://docs.google.com/document/d/1Zr6qzICStrrJwYkfoIE7fTf9Ziu3f6JmhdA9Ao3Y8Bo/edit#heading=h.b7doxousxooq) has the same screenshot. Do you have any thoughts on whether this requires an action?

Reply:
Yeah I think this is the expected screen if there isn't any spark history. If you run some Spark jobs it should be populated.